packages/lib/src/interactions/utils.ts

```diff
@@ -66,24 +65,25 @@ export function getEnclosedRectangle(startPoint: Vector2, endPoint: Vector2) {
 }
 
 export function clampPositionToArea(
-  center: Vector2,
+  center: Vector3,
```

Comment:
Just an idea: it may also be useful to communicate in which space these functions are supposed to act.

Reply:
Good call :+1: 

I was planning on refactoring these utilities anyway, so I'll keep this for another PR.